homography_matrix:
    world2cam: [-1076.85,677.839,-97.8148,-543.74,26.5117,-240.371,-1.65943,0.0135395,-0.154656]
    cam2world: [-4.01363e-06,0.00049125,-0.760979,0.00149399,2.0045e-05,-0.976056,0.000173859,-0.00526928,1.61375]
